_config.yml

```diff
@@ -50,7 +50,8 @@ requires_5_3v: "> **Note**: This feature is only available for TinyMCE 5.3 and l
 requires_5_4v: "> **Note**: This feature is only available for TinyMCE 5.4 and later."
 
 requires_jsscwar_230v: "> **Note**: This feature is only available for TinyMCE self-hosted server-side components, version 2.3.0 and later. To check the version of a running service, visit _&#60;domain&#62;_/_&#60;service&#62;_/version. Such as `http://localhost:8080/ephox-hyperlinking/version`."
-
+deprecate_spellchecker: "> **Important**: The TinyMCE **core** Spell Checker plugin (`spellchecker`) was deprecated with the release of TinyMCE 5.4. For details, see [the TinyMCE core Spell Checker plugin deprecation notice](https://www.tiny.cloud/docs/release-notes/release-notes54/#thetinymcecorespellcheckerplugin)."
```

Comment:
Dislike `core` here can we please remove it? Core has a very different meaning, as that's just the core editor without any plugins.

Reply:
s/core/free/g

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1600/commits/661608ce08831e40b62b55c36d8e4047b5b710e8 and https://github.com/tinymce/tinymce-docs/pull/1600/commits/e6d8bd854f40c0d3ed4da4a222a5d02581680794